src/test/java/seedu/address/model/applicant/EmailTest.java

```diff
@@ -0,0 +1,97 @@
+package seedu.address.model.applicant;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
```

Comment:
```suggestion
import static seedu.address.model.applicant.Email.isValidEmail
```

Can consider adding a static import to avoid lengthy repeated calls